tests/scripts/all.sh

```diff
@@ -34,13 +34,26 @@ MEMORY=0
 SHORT=0
 FORCE=0
 
+OPENSSL="openssl"
```

Comment:
I think we should use existing environment variables to override these options, so we don't have to enter the full path for OpenSSL and GNUTLS at the command line, everytime we execute, so can we make the assignment to <tool>_LEGACY dependent on it not already being defined?